packages/pds/src/db/migrations/20230208T222001557Z-user-table-did-pk.ts

```diff
@@ -0,0 +1,101 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable('user_account')
+    .addColumn('did', 'varchar', (col) => col.primaryKey())
+    .addColumn('email', 'varchar', (col) => col.notNull().unique())
+    .addColumn('password', 'varchar', (col) => col.notNull())
+    .addColumn('lastSeenNotifs', 'varchar', (col) => col.notNull())
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addColumn('passwordResetToken', 'varchar')
+    .addColumn('passwordResetGrantedAt', 'varchar')
+    .execute()
+
+  await db
+    .insertInto('user_account')
+    .columns([
+      'did',
+      'email',
+      'password',
+      'lastSeenNotifs',
+      'createdAt',
+      'passwordResetToken',
+      'passwordResetGrantedAt',
+    ])
+    .expression((exp) =>
+      exp
+        .selectFrom('user')
+        .innerJoin('did_handle', 'did_handle.handle', 'user.handle')
+        .select([
+          'did_handle.did',
+          'user.email',
+          'user.password',
+          'user.lastSeenNotifs',
+          'user.createdAt',
+          'user.passwordResetToken',
+          'user.passwordResetGrantedAt',
+        ]),
+    )
+    .execute()
+
+  await db.schema
+    .createIndex('user_account_password_reset_token_idx')
+    .unique()
+    .on('user_account')
+    .column('passwordResetToken')
+    .execute()
+
+  await db.schema.dropIndex('user_password_reset_token_idx').execute()
```

Comment:
I think it's possible postgres will ask us to remove the `user_email_lower_idx` as well.  We might also want to recreate it on `user_account` if the idea is a one-to-one port of the `user` table.